vizro-core/tests/unit/vizro/models/_components/test_graph.py

```diff
@@ -34,6 +36,87 @@ def standard_px_chart_with_title():
     return px.scatter(data_frame="gapminder", x="gdpPercap", y="lifeExp", title="Title")
 
 
```

Comment:
I think these tests belong in `test_types` and `test_base` tests for `to_python`. See already existing `TestModelFieldJSONConfig` and we could also have something that tests a model there with `CapturedCallableProxy`. These tests aren't really anything to do with graph, more to do with `CapturedCallableProxy` itself. 

Edit: I wrote that under the assumption that we would be rolling out the same change to `vm.Action`, `vm.Grid` etc. but maybe that's not the case?